lib/src/form_props.dart

```diff
@@ -46,6 +46,9 @@ class QtyFormProps {
   /// show or hide cursor
   final bool? showCursor;
 
+  /// override text controller of textfield
+  final TextEditingController? textEditingController;
```

Comment:
I think `textEditingController` is quite long name. can you change it into `controller` ?